### ISSUE TITLE
Cumulative emissions don't match the emission bound

### ISSUE THREAD

**Issue by behnam2015** [iiasa/message_ix#183]
I have a scenario, in which a cumulative "bound_emission" is set for an emission type. As I understand, the cumulative bound for emissions should be equal or greater than the average of relaized emissions (i.e., variable "EMISS") in the cumulative years. However, in my case the average of output "EMISS" is much higher than the "bound_emission".
For further information, the scenario has 5-year time steps until 2060 and 10-year thereafter, and the first model year is 2025 (hence, the cumulative years are 2025, 2030, 2035, ..., 2110). I can send you the model/scenario for further investigation if needed.

**Comment by danielhuppmann** [comment]
Thanks @behnam2015 for raising this issue! Please try whether you can replicate the wrong behaviour in a stylised model to verify this is not just due to a data error in your large model. If it’s indeed a bug, please turn the stylised model into a unit test and make a PR with that.

**Comment by behnam2015** [comment]
Thanks @danielhuppmann for the instructions! I made a unit test showing this behavoir and added the [PR # 187](https://github.com/iiasa/message_ix/pull/187). It seems as far as the model years have equal intervals, either 10 or 5 year, the cumulative bound is calculated correctly. I showed a case that the model years are transiting from 5 year to 10 year, and there the cumulative bound seems not to be working as expected.
